sormas-ui/src/main/java/de/symeda/sormas/ui/map/LeafletMap.java

```diff
@@ -88,6 +97,16 @@ public void call(JsonArray arguments) {
 		String attribution = FacadeProvider.getGeoShapeProvider().loadShapefileAttributions();
 		this.addShapefileAttribution(attribution);
 
+		/*
```

Comment:
This probably makes more sense to be added in the java doc of the constructor or class

Reply:
I'l delete it. Just pasted here to copy parts that were needed for testing